test/integration/app-signed-request-integration.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai'
+import { before, describe, test } from 'mocha'
+import {
+  initClient,
+  initPlainClient,
+  createTestEnvironment,
+  getTestOrganization,
+  createTestSpace,
+} from '../helpers'
+
+describe('AppSignedRequest api', function () {
+  let appDefinition
+  let environment
+  let space
+  let client
+
+  before(async () => {
```

Comment:
Please add an `after` hook to clean up everything

- Uninstall App
- Remove Environment
- Delete Space
- Delete AppDefinition